### DISCUSSION TITLE
404 Error

### DISCUSSION THREAD

**Discussion by Swrice** [fuma-nama/fumadocs#751] [Q&A] [answered]
I continually get a 404 error when trying to go to the /docs url.  All I've done is add logo on home screen and create a meta.json file in the content folder.  it was all working then randomly stopped, even if I delete the meta.json file it will not go back to working.  Currently just working with it on local.

**Comment by fuma-nama** [comment]
Please provide sufficient information for debugging, minimal reproduction is required.

**Reply by Swrice** [reply]
I'm not a coder... so will do my best.  here are screenshots of all the changes I have made in the app.  i can upload the entire app folder if that is helpful?  Sorry for the lack of knowledge, I was trying to build this out without a dev's help.

1. Added to paths in tsconfig.json to get logo defined 
![image](https://github.com/user-attachments/assets/af08e5a2-51c5-46fe-b6db-ca6188d4752f)

2. Adjusted the home screen a bit
![image](https://github.com/user-attachments/assets/19e0b7c1-14f6-4d44-b45b-74391e6a27b4)

3. Added a meta.json file in the content/docs folder 
![image](https://github.com/user-attachments/assets/f53b2b30-7eb5-47b8-8dd9-bc07c03fd258)

**Reply by fuma-nama** [reply]
You don't have a index page here, /docs of course will give you 404

**Reply by Swrice** [reply]
thanks, I'll try and figure it out.  I just did the auto install and assumed it would work.. which it did for a little while.  Appreciate your help, thanks.

**Reply by fuma-nama** [reply, marked as answer]
Add an index.mdx page

**Reply by Swrice** [reply]
okay, I got it now... I had renamed that file, I did not know it was needed. Thanks for the help, it is working now.
